optuna/integration/pytorch_distributed.py

```diff
@@ -91,20 +148,24 @@ def func() -> float:
         return self._call_and_communicate(func, torch.float)
 
     @deprecated("3.0.0", "6.0.0", text=_suggest_deprecated_msg)
+    @updates_properties
```

Comment:
Since this method calls `suggest_float` that is decorated by `updates_properties`, can we remove `updates_properties`?
```suggestion
```

I suppose the other deprecated `suggest_*` methods too.

Reply:
Yes it is possible but I though if all methods are decorated one can change the implementation without having to remember about adding it.

Reply:
Thanks. Hmm, but in my understanding, when we call one of these deprecated suggest methods, `updates_properties` is executed twice, which is supposed to be unnecessary communications among processes. 

@toshihikoyanase  What do you think about this?

Reply:
Good catch! `suggest_uniform` simply calls `suggest_float` and `suggest_float` actually update the properties. So, @nzw0301  suggestion sounds reasonable to me.

> one can change the implementation without having to remember about adding it.

That makes sense, but we will not update the deprecated methods so frequently.

Reply:
I have removed the redundant decorators in the latest commit.